### ISSUE TITLE
docs: Self signed certificate fails on Ubuntu 16.04

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio#3903]
https://github.com/minio/minio/tree/master/docs/tls#with-openssl

## Expected Behavior
Certificates are generated properly

## Current Behavior

```
$ openssl genrsa -out private.key 2048
Generating RSA private key, 2048 bit long modulus
...................+++
......+++
e is 65537 (0x10001)

$ openssl req -new -x509 -days 3650 -key private.key -out public.crt -subj "/C=country/ST=state/L=location/O=organization/CN=domain"
problems making Certificate Request
140255692420760:error:0D07A097:asn1 encoding routines:ASN1_mbstring_ncopy:string too long:a_mbstr.c:158:maxsize=2
```

## Possible Solution
Country name should be 

```
$ openssl req -new -x509 -days 3650 -key private.key -out public.crt -subj "/C=US/ST=state/L=location/O=organization/CN=domain"
```

## Steps to Reproduce (for bugs)

Just follow the referred document link.

## Context
To be able to configure SSL

## Your Environment
All versions.

**Comment by koolhead17** [comment]
This can be closed via https://github.com/minio/minio/pull/3957

**Comment by harshavardhana** [comment]
Fixed by #3957
